package.json

```diff
@@ -131,12 +131,20 @@
       }
     },
     "rules": {
-      "@typescript-eslint/no-unused-vars": [
+      "import/no-anonymous-default-export": [
         1,
         {
-          "vars": "local"
+          "allowObject": true
         }
       ],
```

Comment:
When i updated eslint, we started getting errors in our stories where it does

```jsx
export default { Title: 'Components/AccessibleIcon' };
```

This rule change allows default object exports.